script-library/powershell-debian.sh

```diff
@@ -18,19 +18,25 @@ fi
 
 export DEBIAN_FRONTEND=noninteractive
 
-# Install curl, apt-transport-https, lsb-release, or gpg if missing
+# Install curl, apt-transport-https, or gpg if missing
 if ! dpkg -s apt-transport-https curl ca-certificates lsb-release > /dev/null 2>&1 || ! type gpg > /dev/null 2>&1; then
```

Comment:
`lsb-release` was removed from the comment, but not `if` condition

Reply:
Ack - gnupg2 needs to be moved up too.